### DISCUSSION TITLE
Option for Zen Sidebar to be permanently visible, but collapsed.

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#865] [Ideas]
### What feature would you like to see?

Before discovering the lovely Zen Browser only comparatively recently i've been using Floorp. It's been tricky for me to decide which one to make my default, as they're both great, despite also having differences. On balance, so far, i prefer Floorp's design for the **dedicated Web Panels Sidebar**, being similar to the venerable Vivaldi & Opera Presto designs, in that it's **always visible**, albeit collapsed. I use Web Panels a lot, & in Floorp i can get to them faster than in Zen, which needs some extra clicks. 

My request is for it to be an extra _option_, not a mandatory replacement for the current Zen method, as i appreciate that some users will prefer that. 

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/293_

**Comment by Dehelssey** [comment]
I'm wanting this so much as well !

**Comment by SilentUK90** [comment]
+1 this would be the best feature added to zen along with tab groups imo.  This is the only thing preventing me from moving over from Floorp 100%

**Comment by guigirl42** [comment]
The Dev has this current/recent reddit thread; https://libreddit.oxymagnesium.com/r/zen_browser/comments/1fk0x80/new_possible_sidebar_ui_looking_for_feedback/. I have no reddit account so cannot comment there, but will do so here in this Github Issue, given i raised its spawning thread https://github.com/zen-browser/desktop/issues/293. 

If the Zen Sidebar is being redesigned, i would _really really really_ hope this redesign provides an _optional_ feature for users who want it, to move the `Web Side Panels` from being hidden away as they are now, to being available as a vertical bar [visible or autohidden] on either the LHS or RHS of the browser UI [like Floorp & Vivaldi]. Please.

**Comment by vraravam** [comment]
I also would love for the sidebar to be "collapsed" (ie only icons) unless I hover on the sidebar itself. Upon hovering, it can expand to show the tab title. (I think this is how it used to be in the alpha.19 build, but the twilight builds have the sidebar in an "always show" mode with a min/max width. Please allow for the user's choice to override and collapse

**Comment by jquintanilla4** [comment]
+1 on this. A mixture of single toolbar and collapsed sidebar would be the perfect mix.

**Comment by SilentUK90** [comment]
Something like this would be perfect 

https://github.com/aminought/firefox-second-sidebar

**Reply by guigirl42** [reply]
Agree! Since i discovered that project one or two months ago, & installed it in my `Firefox Nightly` [which already runs `Sidebery` for the best tabs management paradigm in existence], I no longer need to use `Zen Browser` [at least, not until it stabilises & stops having such enormous design changes so frequently]. Whilst i appreciate that Zen gives us `Web Panels`, i have disliked from the beginning the _extra_ clicks required to access & use them, compared to a powerful always-visible dedicated second sidebar for WPs such as used in `Floorp` & `Vivaldi`. The excellent `Firefox Second Sidebar` project fills the gap beautifully. I do wish the Zen Dev would countenance a similar design philosophy change.

**Comment by pjb-tech** [comment]
> ### What feature would you like to see?
> 
> Before discovering the lovely Zen Browser only comparatively recently i've been using Floorp. It's been tricky for me to decide which one to make my default, as they're both great, despite also having differences. On balance, so far, i prefer Floorp's design for the **dedicated Web Panels Sidebar**, being similar to the venerable Vivaldi & Opera Presto designs, in that it's **always visible**, albeit collapsed. I use Web Panels a lot, & in Floorp i can get to them faster than in Zen, which needs some extra clicks.
> 
> My request is for it to be an extra _option_, not a mandatory replacement for the current Zen method, as i appreciate that some users will prefer that.
> ### What platform are you seeing the problem on?
> 
> Linux
> ### Additional context
> 
> _No response_
> 
> ~ _Created automatically from issue: #293_

Bump, because I would love this.
